src/rapidoc.js

```diff
@@ -566,6 +566,19 @@ export default class RapiDoc extends LitElement {
     }
   }
 
+  resetDoc() {
+    this.resolvedSpec = null;
+    this.loading = true;
+    this.loadFailed = false;
+    this.requestUpdate();
+
+    // Remove the previous active tag
```

Comment:
NavBar is available in `read` and `focused` mode only, so check if `renderStyle` is `focused` or `read` then only perform these statements

Reply:
Updated PR